app/presenters/redirect_presenter.rb

```diff
@@ -26,8 +26,8 @@ def for_message_queue(payload_version)
     )
   end
 
-  def for_redirect_helper(content_id)
-    present.merge(content_id: content_id)
+  def for_redirect_helper(content_id, update_type)
+    present.merge(content_id: content_id, update_type: update_type)
```

Comment:
I'd probably just include a hard coded update_type of major here. 
I think having a variety of update_types on redirects is a bit confusing (well the whole concept of an update_type on a redirect is confusing).

This would also be consistent with how they are currently published: https://github.com/alphagov/publishing-api/blob/master/app/commands/v2/publish.rb#L186

And I also wonder once we add this in if we can remove the line I linked to above?

Reply:
This has now been updated 🌞